.github/workflows/build.yml

```diff
@@ -1,7 +1,7 @@
 on:
   push:
     branches:
-      - master
+      - "*"
```

Comment:
we probably want to revert this change

Reply:
I don't mind building a new image for every branch, so people can test their changes, WDYT?

I'll create an issue to do a periodic cleanup of old dev images.

Reply:
Ah, ok. Yeah, that may be useful. So it can be even simplified to `on: push`